### ISSUE TITLE
Energy history pages cannot be queried via the bridge mode

### ISSUE THREAD

**Issue by jfhautenauven** [robertklep/bosch-xmpp#29]
Hi Robert, me again :) 

Just to let you know, i'm trying to access the energy data from the bosch easycontrol I got here.
Got two URL to do so : 

/bridge/energy/history and /bridge/energy/historyEntries

Historically, we used to access the energy history by adding ?page=1 , 2 etc.
Doesnt seem to work via the bridge mode.

Any clue ? Do i do it wrong or is this an issue?

KR,
JFH

**Comment by jfhautenauven** [comment]
Hi Robert, sadly, i've tested the latest version, but I cannot get the syntax "?page=x" to work.

I can put whatever I want, even "?helloWorld=69", it doesnt change anything. It ignores it.
Can you have a check ?

Thanks in advance

**Comment by robertklep** [comment]
Seems to work fine for me:
```
$ curl 'localhost:8011/bridge/ecus/rrc/recordings/gasusage?page=20'
{"id":"/ecus/rrc/recordings/gasusage","type":"recordings","recordable": … }
```

Do make sure you escape question marks on the command line to prevent the shell from interpreting them.

**Comment by jfhautenauven** [comment]
I try accessing it from a browser.
But the command you give is actually for an old thermostat (IVT ? )
For the EasyControl, I see two paths : 

http://10.0.0.180:3000/bridge/energy/history
http://10.0.0.180:3000/bridge/energy/historyEntries

with the following results

![image](https://user-images.githubusercontent.com/83645528/155131043-e7fc8be4-bab7-4dfc-bf68-6dc83a128ee3.png)

![image](https://user-images.githubusercontent.com/83645528/155131098-fd98c8d0-0ad1-4786-a0d2-d608512b16f2.png)


On the first, URL, I get a subset of the history that there should be.
http://10.0.0.180:3000/bridge/energy/history and http://10.0.0.180:3000/bridge/energy/history?page=2 give exactly the same result, nothing happens.


I tried with curl also, same result, escaping etc is ok. :/

**Comment by robertklep** [comment]
I'm testing against a Nefit Easy thermostat.

Do your endpoints work as expected with the commandline tool?
```
$ bosch-xmpp … get '/energy/history?page=2'
```

**Comment by robertklep** [comment]
Also, you can run the bridge in debug mode to see which queries it sends to the backend:
```
$ env DEBUG=* bosch-xmpp … easycontrol bridge 
```

**Comment by jfhautenauven** [comment]
Not working as expected, I always get the first page

**Comment by robertklep** [comment]
If it doesn't work as expected with the commandline tool, I have to assume that pagination just works differently on the EasyControl (which I don't have, so can't test).

**Comment by robertklep** [comment]
Ah, it should be "entry", not "page".

See [this comment](https://github.com/robertklep/bosch-xmpp/issues/17#issuecomment-855273735).

**Comment by jfhautenauven** [comment]
Confirmed Robert, with "entry" it works.
You're the best :D

**Comment by robertklep** [comment]
I released it as v1.4.0 😊

**Comment by jfhautenauven** [comment]
awesome thanks Robert :D
